### PR TITLE
Add notice for Field Conditions bug

### PR DESCRIPTION
**Pull request by paescuj**

### CODE REVIEW

File: .changeset/fast-ways-approve.md [hunk 1]
```diff
@@ -2,4 +2,12 @@
 "@directus/app": patch
 ---
 
+::: notice
+
+Due to a bug introduced in v10.6.2, Field Conditions were applied incorrectly. This has been corrected and works again as expected.
```
Comment (paescuj) [inline .changeset/fast-ways-approve.md:7]: Suggestion from @br41nslug ❤️ 
```suggestion
Due to a bug introduced in v10.6.2, Field Conditions were saved incorrectly. This has been corrected and works again as expected.
```

File: .changeset/fast-ways-approve.md [hunk 2]
```diff
@@ -2,4 +2,12 @@
 "@directus/app": patch
 ---
 
+::: notice
+
+Due to a bug introduced in v10.6.2, Field Conditions were saved incorrectly. This has been corrected and works again as expected.
+If you have edited Field Conditions under this version, please refer to https://github.com/directus/directus/issues/19757#issuecomment-1734162756 for instructions to fix the Conditions.
```
Comment (paescuj) [inline .changeset/fast-ways-approve.md:8]:
```suggestion
However, if Field Conditions have been edited under this version, they need to be corrected. Please refer to https://github.com/directus/directus/issues/19757#issuecomment-1734162756 for instructions on how to do so.
```

**Review by br41nslug — APPROVED** [review]
Awesome, at least we can point people to somewhere if they're running into this issue 😄

File: .changeset/fast-ways-approve.md [hunk 3]
```diff
@@ -2,4 +2,12 @@
 "@directus/app": patch
 ---
 
+::: notice
+
+Due to a bug introduced in v10.6.2, Field Conditions were saved incorrectly. This has been corrected and works again as expected.
```
Comment (paescuj) [inline .changeset/fast-ways-approve.md:7]:
```suggestion
Due to a bug introduced in v10.6.2, Field Conditions were saved incorrectly. This has been fixed and works again as expected.
```

File: .changeset/fast-ways-approve.md [hunk 4]
```diff
@@ -2,4 +2,12 @@
 "@directus/app": patch
 ---
 
+::: notice
+
+Due to a bug introduced in v10.6.2, Field Conditions were saved incorrectly. This has been fixed and works again as expected.
+However, if Field Conditions have been edited under this version, they need to be corrected. Please refer to https://github.com/directus/directus/issues/19757#issuecomment-1734162756 for instructions on how to do so.
```
Comment (paescuj) [inline .changeset/fast-ways-approve.md:8]: A.k.a the last change 😄 
```suggestion
However, if Field Conditions have been edited under this version, they need to be corrected. Please refer to https://github.com/directus/directus/issues/19757#issuecomment-1734162756 for instructions.
```

**Comment by br41nslug** [comment]
Rechecked the text and my approval stands 😄

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**